.github/workflows/sob.yml

```diff
@@ -0,0 +1,9 @@
+name: DCO
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  dco-check:
+    uses: cncf/dcochecker/.github/workflows/dco.yml@main
```

Comment:
I am paranoid on adding non-standard stuff into the project. Looking into https://github.com/cncf/dcochecker I am not sure how active it is supported. Besides, we can implement this with just something like `git -1 | grep Signed-off-by`, no need in 3rd party action.

Reply:
Just double-checking: By "non-standard" you mean, actions done by other parties - not by Github themselves. correct?

Sure it can be re-implemented locally - can give it a try unless someone beats be to it.

Reply:
> actions done by other parties - not by Github themselves. correct?

Yes. Everything outside of Github itself would need careful consideration. It's not impossible to take them, but we need a confidence these would be supported for a really long period of time.